Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -7247,6 +7265,2013 @@ def panorama_install_file_content_update_command(args: dict):
+
+
+@dataclass
+class ShowCounterGlobalResultData(ResultData):
+    """
+    :param category: The counter category
+    :param name: Human readable counter name
+    :param value: Current counter value
+    :param rate: Packets per second rate
+    :param aspect: PANOS Aspect
+    :param desc: Human readable counter description
+    :param counter_id: Counter ID
+    :param severity: Counter severity
+    :param id: Counter ID
+    """
+    category: str
+    name: str
+    value: int
+    rate: int
+    aspect: str
+    desc: str
+    id: str
+    severity: str
+
+    timestamp = datetime.now()
+
+    def __post_init__(self):
+        self.value = int(self.value)
+        self.rate = int(self.rate)
+
+
+@dataclass
+class ShowCounterGlobalSummaryData(ResultData):
+    """
+    :param name: Human readable counter name
+    :param value: Current counter value
+    :param rate: Packets per second rate
+    :param desc: Human readable counter description
+    """
+    name: str
+    value: int
+    rate: int
+    desc: str
+
+    def __post_init__(self):
+        self.value = int(self.value)
+        self.rate = int(self.rate)
+
+
+@dataclass
+class ShowCounterGlobalCommmandResult:
+    summary_data: List[ShowCounterGlobalSummaryData]
+    result_data: List[ShowCounterGlobalResultData]
+
+    _output_prefix = OUTPUT_PREFIX + "ShowCounters"
+    _title = "PAN-OS Global Counters"
+
+    _summary_cls = ShowCounterGlobalSummaryData
+    _result_cls = ShowCounterGlobalResultData
+
+
+@dataclass
+class ShowRoutingProtocolBGPPeersResultData(ResultData):
+    """
+    :param peer: Name of BGP peer
+    :param vr: Virtual router peer resides in
+    :param remote_as: Remote AS (Autonomous System) of Peer
+    :param status: Peer connection status
+    :param incoming_total: Total incoming routes from peer
+    :param incoming_accepted: Total accepted routes from peer
+    :param incoming_rejected: Total rejected routes from peer
+    :param policy_rejected: Total routes rejected by peer by policy
+    :param outgoing_total: Total routes advertised to peer
+    :param outgoing_advertised: Count of advertised routes to peer
+    :param peer_address: IP address and port of peer
+    :param local_address: Local router address and port
+    """
+    peer: str
+    vr: str
+    remote_as: str
+    status: str
+    peer_address: str
+    local_address: str
+    incoming_total: int = 0
+    incoming_accepted: int = 0
+    incoming_rejected: int = 0
+    policy_rejected: int = 0
+    outgoing_total: int = 0
+    outgoing_advertised: int = 0
+
+    def __post_init__(self):
+        self.incoming_total = int(self.incoming_total)
+        self.incoming_accepted = int(self.incoming_accepted)
+        self.incoming_rejected = int(self.incoming_rejected)
+        self.policy_rejected = int(self.policy_rejected)
+        self.outgoing_total = int(self.outgoing_total)
+        self.outgoing_advertised = int(self.outgoing_advertised)
```

Comment:
why do you need to cast those into int if they are already ints?

Reply:
When they get deserialized from XML, they are strings. Instead of handling it at deserialization, we do it here, this way it's impossible to create an instance of this dataclass where those values are incorrectly string ints ("1")